docs/usage/configuration-templates.md

```diff
@@ -5,57 +5,66 @@ description: How to edit Renovate's config templates
+You're probably fine leaving this setting alone, though you can change it.
+e.g. if you prefer that Renovate uses the term 'Upgrade' instead of 'Update' then you could configure `"commitMessageAction": "Upgrade"`.
 
-`commitMessageTopic` is usually 2-3 words aimed to identify _what_ is being updated. e.g. it might be `dependency react` or `Docker image ubuntu`.
-You may want to edit this, but if you think your idea/requirement is a good one then maybe you can propose it to the project or publish it as a preset config for others with similar requirements.
+`commitMessageTopic` is usually two to three words that identify _what_ is being updated.
+e.g. it might be `dependency react` or `Docker image ubuntu`.
+You may want to edit this.
+If you think your new `commitMessageTopic` is helpful for others, please [open a PR](https://github.com/renovatebot/renovate/pulls).
 
-`commitMessageExtra` usually refers to the version being updated to. e.g. `to v16` for a major upgrade, or `to v16.0.3` for a patch update.
-It may also be empty in some cases, e.g. if the action/topic is `Pin Docker digests`.
+`commitMessageExtra` refers to the version being updated to.
+e.g. `to v16` for a major upgrade, or `to v16.0.3` for a patch update.
+It can be empty in some cases, like if the action/topic doesn't change a package version, e.g. `Pin Docker digests`.
 
-`commitMessageSuffix` defaults to empty and is there for flexibility and future use.
-Maybe for `major` updates you always want the PR to end with `(MAJOR)`, for instance.
+`commitMessageSuffix` defaults to empty but is currently used in two cases:
+
+- Differentiating major from non-major groups
+- Differentiating between PRs from different base branches, maybe for `major` updates you always want the PR to end with `(MAJOR)`, for instance
 
 `commitBody` is used if you wish to add multi-line commit messages, such as for the `Signed-off-by` fields, or adding `[skip-ci]`, etc.
 It is appended to the generated `commitMessage`, separated by a newline.
 
 ## PR Title
 
-Because commit messages should usually match with the PR title, the PR title template now defaults to `null` and inherits whatever is configured for `commitMessage`.
-If you have a requirement where `prTitle` should be different from `commitMessage`, then please raise a feature request for discussion.
+Because commit messages match with the PR title, the PR title template defaults to `null` and inherits/copies the value from `commitMessage`.
+If you have a requirement where `prTitle` should be different from `commitMessage`, then please [raise a feature request](https://github.com/renovatebot/renovate/issues) for discussion.
 
 ## PR Body
 
 The PR Body is currently a little difficult to edit because of its size, however it should soon be redesigned like the other templates to allow more easier editing without needing to copy/paste the whole template.
+TODO: CHECK WITH RENOVATE TEAM IF THE PR BODY IS USER EDITABLE NOW
```

Comment:
```suggestion
The PR body can be edited multiple ways:
- Editing the entire layout/flow using `prBodyTemplate` (but try to avoid that if possible)
- Adding header or footer using `prHeader` and `prFooter`
- Adding notes using `prBodyNotes`
- Editing the embedded table using `prBodyDefinitions` and `prBodyColumns`
```